src/Antiforgery/src/Internal/DefaultClaimUidExtractor.cs

```diff
@@ -119,33 +111,40 @@ public DefaultClaimUidExtractor(ObjectPool<AntiforgerySerializationContext> pool
         return identifierParameters;
     }
 
-    private byte[] ComputeSha256(IEnumerable<string> parameters)
+    private static void ComputeSha256(IList<string> parameters, Span<byte> destination)
```

Comment:
```suggestion
    private static void ComputeSha256(List<string> parameters, Span<byte> destination)
```

Use concrete types for internal methods.